### ISSUE TITLE
Issue with Databricks absolute path tables

### ISSUE THREAD

**Issue by Fokko** [sqlfluff/sqlfluff#602]
We're using DBT with Databricks and tried to implement sqlfluff for linting. However, we're seeing that the following syntax is unsupported:

```sql
SELECT *
FROM delta.`/mnt/datalake/table`
```

Is this something that we want to support in sqlfluff? If so, I can try to come up with a PR.

**Comment by dmateusp** [comment]
hi @Fokko, glad to see you here!

sqlfluff has a modular approach to dialects, you should be able to see all the dialects here: https://github.com/sqlfluff/sqlfluff/tree/master/src/sqlfluff/core/dialects

I'm thinking you would need to implement a Spark SQL dialect (which would be awesome to have)

Related #417

**Comment by R7L208** [comment]
This is not specific to Databricks but the Delta Lake OSS project. The Delta dialect should probably be an extension of Spark SQL and a valid file format in DBT and HQL.

https://docs.delta.io/latest/delta-batch.html#read-a-table

https://github.com/delta-io/connectors/#create-a-hive-table
https://docs.getdbt.com/reference/resource-configs/spark-configs#configuring-tables

**Comment by R7L208** [comment]
@tunetheweb - Can you change the label from `new dialect` to `spark3`?

Spark docs have been updated recently. This should be supported in Spark dialect then Databricks (which is SaaS Spark) could inherit it.

https://spark.apache.org/docs/latest/sql-ref-syntax-qry-select-file.html

**Comment by tunetheweb** [comment]
> @tunetheweb - Can you change the label from `new dialect` to `spark3`?

Done!
